infra/README.md

```diff
@@ -77,17 +86,22 @@ Instructions:
    # If set to true, pull the base ubuntu image from Artifact Registry.
    # Otherwise, assumes GCR.
    use_artifact_registry = false
+
```

Comment:
I can't comment above, but do we need the requester pays bucket up here?

Reply:
I reuse the `hail_test_gcs_bucket_*` variables as idk why we would want a different configuration for the test bucket vs the test requester pays bucket.